.github/workflows/pipeline_v3_ci.yml

```
@@ -0,0 +1,262 @@
+name: Pipeline v3 CI
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)